src/legacy/core_plugins/vis_type_tagcloud/public/types.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+export interface TagCloudVisParams {
```

Comment:
For visualizations that have been converted to TS, many of the vis params are already typed in the interpreter function file... So in the case of tagcloud, there's already an `Arguments` interface in `tag_cloud_fn.ts` which extends this.

Your types here are much better, so if you are planning to implement these interfaces, would you mind also updating the relevant `*_fn.ts` files (if present) so that we aren't duplicating types there and are instead importing from this location?